sql/updates/post-update.sql

```diff
@@ -171,3 +171,40 @@ $$;
 -- Repair relations that have relacl entries for users that do not
 -- exist in pg_authid
 CALL _timescaledb_functions.repair_relation_acls();
+
+-- Migrate existing CAggs using time_bucket_ng to time_bucket
+CREATE OR REPLACE PROCEDURE _timescaledb_functions.cagg_migrate_to_time_bucket(cagg REGCLASS)
+   AS '@MODULE_PATHNAME@', 'ts_continuous_agg_migrate_to_time_bucket' LANGUAGE C;
```

Comment:
Is this a function we create temporarily just for this cleanup? In that case, should it be dropped later?

Reply:
Ahh this function exists only starting on 2.15.0 and it is not temporary